packages/performance-tests/README.md

```diff
@@ -0,0 +1,51 @@
+# Presentation Performance Tests
+
+A package containing performance tests for the [`@itwin/transformer` library](../../README.md).
+
+## Tests
+
+Here are tests we need but don't have:
+
+- *Identity Transform*
+  transform the entire contents of the iModel to an empty iModel seed
+- *JSON Geometry Editing Transform*
+  transform the iModel, editing geometry as we go using the json format
+- *Binary Geometry Editing Transform*
+  transform the iModel, editing geometry as we go using elementGeometryBuilderParams
+- *Optimistically Locking Remote Target*
+- *Pessimistically Locking Remote Target*
+- *Processing Changes*
+- *More Branching Stuff*
+
+<!-- FIXME: this isn't true yet-->
+All tests run in isolation on every iModel in these projects:
+
+- https://qa-connect-imodelhubwebsite.bentley.com/Context/892aa2c9-5be8-4865-9f37-7d4c7e75ebbf
```

Comment:
remove these references to internal imodels. They should be configured in an .env file or something, that we can set in our internal pipelines, and external users should also be able to configure their own imodels for these tests.